src/main/webapp/index.html

```diff
@@ -70,4 +61,7 @@
       </div>
     </nav>
   </body>
+  
+  <!-- JavaScript -->
+  <script src="script.js"></script>
```

Comment:
Is there any specific reason to link the script file here and not inside the head tag?

Reply:
Yes! I was having a bug where I needed to insert data into a div inside of the body, but because the script was in the header, and the function was called on the "onload" of the body, the div had not yet loaded. The only solution I found to this issue, is have the script moved to the bottom of the HTML, so that all of the content of the body had already been loaded. I've worked on other projects where the scripts are at the bottom as well, so I think it's not a problem.

Reply:
Makes sense!